cloudbuild/cloudbuild.yaml

```diff
@@ -1,16 +1,24 @@
 steps:
 
+- id: Fetch service account key
+  name: gcr.io/cloud-builders/gsutil
+  args:
+  - cp
+  - gs://cloud-marketplace-ops-test-kokoro/application_credentials
```

Comment:
We also have [this service account](https://storage.cloud.google.com/cloud-marketplace-ops-test-kokoro/cloud-marketplace-ops-test-5cb3e979a3ae.json) that we use in our kokoro builds. This `application_credentials` seems new--what are the special permissions here?